src/pages/stablecoins.js

```diff
@@ -249,8 +249,15 @@ const InfoIcon = styled(Icon)`
   fill: ${(props) => props.theme.colors.text};
 `
 
-const ButtonLinkRight = styled(ButtonLink)`
-  margin-left: 1rem;
+const ButtonLinkWrap = styled(ButtonLink)`
+  white-space: break-spaces;
```

Comment:
```suggestion
    width: 100%;
    text-align: left;
    white-space: inherit;
```
This definitely fixes the problem, but I think these suggested styles look a bit better in my opinion. That way we aren't mixing text alignments. Let me know what you think @vluna

Reply:
I am not sure this will work for desktop since the button might use 100% width. We could do `white-space: inherit` only, it will produce the same result. However, I was thinking that maybe we should change the `white-space: nowrap` to  `inherit` in the ButtonLink component as this might happen to all the buttons. With `inherit`, it will just break to the next line by default instead of going all the way. 

I believe the `white-space` change should be sufficient when this issue comes up. What do you think @corwintines?